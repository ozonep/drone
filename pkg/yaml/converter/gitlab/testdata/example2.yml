before_script:
  - bundle install

test2.1:
  image: ruby:2.1
  services:
  - postgres:9.3
  script:
  - bundle exec rake spec

test2.2:
  image: ruby:2.2
  services:
  - postgres:9.4
  script:
  - bundle exec rake spec