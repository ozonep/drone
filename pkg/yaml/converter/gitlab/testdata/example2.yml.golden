---
kind: pipeline
name: test2.1

platform:
  os: linux
  arch: amd64

steps:
- name: test2.1
  image: ruby:2.1
  commands:
  - bundle install
  - bundle exec rake spec

---
kind: pipeline
name: test2.2

platform:
  os: linux
  arch: amd64

steps:
- name: test2.2
  image: ruby:2.2
  commands:
  - bundle install
  - bundle exec rake spec

depends_on:
- test2.1

...
